sdk/core/Azure.Core.TestFramework/src/RecordMatcher.cs

```diff
@@ -123,9 +123,9 @@ public virtual RecordEntry FindMatch(RecordEntry request, IList<RecordEntry> ent
             throw new TestRecordingMismatchException(GenerateException(request, bestScoreEntry));
         }
 
-        private int CompareBodies(byte[] requestBody, byte[] responseBody, StringBuilder descriptionBuilder = null)
+        private int CalculateCompareBodies(byte[] requestBody, byte[] responseBody, StringBuilder descriptionBuilder = null)
```

Comment:
Why calculate?

Reply:
Conflict with the CompareBodies property that was added.